tests/LogTargetTest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace yiiunit\extensions\debug;
+
+use yii\debug\LogTarget;
+use yii\debug\Module;
+
+class LogTargetTest extends TestCase
+{
+    public function testGetRequestTime()
+    {
+        $logTarget = new LogTarget(new Module('debug'));
+        $actual = $this->invoke($logTarget, 'getRequestTime');
+        $this->assertTrue(is_float($actual));
```

Comment:
the point of this test is that you are testing a float, whatever it's source might be, the whole point of this commit.